src/compiler/config/validate-paths.ts

```diff
@@ -86,7 +86,7 @@ export function validatePaths(config: Config) {
     config.wwwIndexHtml = normalizePath(path.join(config.wwwDir, DEFAULT_INDEX_HTML));
   }
   if (!path.isAbsolute(config.wwwIndexHtml)) {
-    config.wwwIndexHtml = normalizePath(path.join(config.rootDir, config.wwwDir));
+    config.wwwIndexHtml = normalizePath(path.join(config.wwwDir, config.wwwIndexHtml));
```

Comment:
Awesome thanks for finding this! So one thing we've been doing is that if the file is relative, it should be relative to the stencil.config.js file, so basically the project's root. So in this case, i think it'd be normalizePath(path.join(config.rootDir, config.wwwIndexHtml)); Thoughts?

Reply:
That's totally your call. Looking over the file it's inconsistent what paths are absolutized against. I see build is also against `wwwDir`. Dunno if output files should be relative to `wwwDir` or if these all need a bit of a rename.

Reply:
🤔 very good point! Let's go with it, thanks!